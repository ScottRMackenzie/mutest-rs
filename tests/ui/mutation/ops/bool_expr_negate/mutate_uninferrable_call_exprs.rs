//@ print-mutants
//@ build
//@ stdout
//@ stderr: empty
//@ mutation-operators: bool_expr_negate

fn f() {
    let _: bool = Default::default();
}

#[test]
fn test() {
    f();
}
