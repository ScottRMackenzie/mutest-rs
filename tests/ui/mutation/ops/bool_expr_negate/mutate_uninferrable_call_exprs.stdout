[bool_expr_negate] negate boolean expression in f at tests/ui/mutation/ops/bool_expr_negate/mutate_uninferrable_call_exprs.rs:8:19: 8:37
  <-(0)- test

1 mutations; 1 safe; 0 unsafe (0 tainted)
